web/themes/custom/usagov/usagov.theme

```diff
@@ -10,6 +10,32 @@ use Drupal\taxonomy\Entity\Term;
  * Place your custom PHP code in this file.
  */
 
+/**
+ * Some Spanish Basic-Pages in the system for some reason do not load by their alias paths,
+ * so in here we will notice when we are on a /node/#### page, and manually lookup the alias
+ * from the path_alias table.
+ */
+function usagov_preprocess_block__usagov_taxonomy_links(&$variables) {
+
+    // In theory, this should return an alias-path, but in certain edge-cases, we get the system path.
+    $urlToPass = \Drupal::request()->getRequestUri();
+
+    // If we are NOT seeing an alias path...
+    if ( substr($urlToPass, 0, 6) === '/node/' ) {
```

Comment:
If we go forward with this, let's log a warning here, so we can look for this kind of thing in our logs and know when we have problems to fix.

Reply:
Sure, just added the warning in (on now-line 27). Not sure if this matters, but should I not label this as a warning from "usagov_ssg_postprocessing"? The line really comes from this theme file, but I know the SSG will be the thing to really pop out the warnings when they do hit